static/src/javascripts/projects/commercial/modules/paid-containers.ts

```diff
@@ -0,0 +1,35 @@
+import bean from 'bean';
```

Comment:
There are rules that will prevent you from using `bean` in TypeScript files. You have to use `lib/$$` instead.

Reply:
Ah is this why teamcity build is breaking? It wasn't clear to me from the logs

Reply:
You should see this rule highlighted in vscode.
There is a flakey test `scroll.spec.js` which fails the build sometimes